src/clients/SpokePoolClient.ts

```diff
@@ -672,6 +673,11 @@ export class SpokePoolClient extends BaseAbstractClient {
       }
     }
 
+    const hubCurrentTime = this.hubPoolClient?.currentTime;
+    if (!isDefined(hubCurrentTime)) {
```

Comment:
Could we short circuit this and directly check `!isDefined(hubCurrentTime)` or is this needed to appease TS?

Reply:
This is mainly to convince ts that hubCurrentTime is a valid number if this if statement isn't hit.

Reply:
SGTM! Just wanted to double check that this was the reason.